hugegraph-test/src/main/java/com/baidu/hugegraph/unit/cassandra/CassandraTest.java

```diff
@@ -42,7 +41,7 @@ public class CassandraTest {
     @Before
     public void setup() {
         OptionSpace.register("cassandra",
-        "com.baidu.hugegraph.backend.store.cassandra.CassandraOptions");
+            "com.baidu.hugegraph.backend.store.cassandra.CassandraOptions");
     }
 
     @After
```

Comment:
can you align such as 
![5](https://user-images.githubusercontent.com/13429553/167639578-e92574a0-450f-4460-bb10-f10465bcd8da.jpg)